Client/Frontend/Browser/ClipboardBarDisplayHandler.swift

```diff
@@ -0,0 +1,82 @@
+
+import Foundation
+import Shared
+
+public struct UserDefaultClipboardKey {
+    public static let KeyLastSavedURL = "KeylastSavedURL"
+}
+
+public struct ClipboardBarToastUX {
+    static let DismissAfter = 10.0
+}
+
+protocol ClipboardBarDisplayHandlerDelegate: class {
+    func shouldDisplayClipboardBar(absoluteString: String)
+}
+
+class ClipboardBarDisplayHandler {
+    weak var delegate: ClipboardBarDisplayHandlerDelegate?
+    private var sessionStarted = true
+    private var prefs: Prefs
+    private var lastDisplayedURL: String? {
+        if let value = UserDefaults.standard.object(forKey: UserDefaultClipboardKey.KeyLastSavedURL) as? String {
```

Comment:
I don't think this will be wiped when a user clears their history will it? This is adding some complexity to this feature. What if you just store this in memory so if a user switches apps the toast doesnt keep reappearing. But it'll reappear after a while if the app restarts. Or if we just lower how long the toast appears maybe we can just show it every time.